File: baekjoon_all/17000+/boj_17094.py
# Solve 2023-12-21

import sys
from collections import Counter

input = lambda : sys.stdin.readline().rstrip()

_ = input()
s = input()
cnt = Counter(s)

if cnt["2"] > cnt["e"]:
    print("2")
elif cnt["2"] < cnt["e"]:
    print("e")
else:
    print("yee")

File: baekjoon_all/17000+/boj_17094.cpp
// Solve 2023-12-21

#include <bits/stdc++.h>
using namespace std;

#define FASTIO ios_base::sync_with_stdio(false);cin.tie(NULL); // boj_15552.cpp
#define SETPRECISION(n) cout << fixed;cout.precision(n); // boj_1008.cpp
#define SIZE(v) (int)v.size()
#define ALL(v) v.begin(),v.end()
using ll = long long;

int main() {
    FASTIO;

    int n;
    string s;
    cin >> n >> s;

    int cnt[2] = { 0 };

    for (char c : s) {
        if (c == '2') cnt[0]++;
        else cnt[1]++;
    }

    if (cnt[0] > cnt[1]) cout << "2\n";
    else if (cnt[0] < cnt[1]) cout << "e\n";
    else cout << "yee\n";

    return 0;
}
